RHTE-LAB.adoc

```diff
@@ -431,11 +443,11 @@ $ *grep SCHEDULE -C10 ./.applier/group_vars/seed-hosts/main.yml*
     namespace: openshift-config
 --------------------------------------------------------------------------------
 
-. Identify where the ldap_cron_schedule is set in seed-hosts Ansible group variables:
+. Identify where the `ldap_cron_schedule` variable is set in seed-hosts Ansible group variables:
 +
 [subs=+quotes]
 --------------------------------------------------------------------------------
-$ **grep ^ldap_cron_schedule -B1 .applier/group_vars/seed-hosts/***
+$ **grep -r ^ldap_cron_schedule -B1 .applier/group_vars/seed-hosts/***
```

Comment:
The command formatted from asciidoc was `grep ^ldap_cron_schedule -B1 .applier/group_vars/seed-hosts/*`, it seems the wildcard was dropped in the version you saw?